Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -22,16 +22,8 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .shippingLabelsOnboardingM1:
             return buildConfig == .localDeveloper || buildConfig == .alpha
-        case .promptToEnableCodInIppOnboarding:
-            return true
         case .searchProductsBySKU:
             return true
-        case .inAppPurchases:
-            return buildConfig == .localDeveloper || buildConfig == .alpha
```

Comment:
As I see, this wasn't part of the release, but now, when removing the feature flag, it will be. Is this the intended behavior? Do we want release notes for this change?

Reply:
Argh thank you for spotting this. I thought it wasn't used, but actually it is, for the debug IAP menu. We don't really want the toggle to turn that debug menu on starting to show up in production builds.

I've put the flag back in 7dc6b4a and I'll let Paolo know that we've left it in place. They may just want to remove that menu, or at least make it permanently only available in debug builds.